libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewStyleCustomization.kt

```diff
@@ -24,7 +26,11 @@ import com.mapbox.navigation.ui.voice.view.MapboxAudioGuidanceButton
  */
 @ExperimentalPreviewMapboxNavigationAPI
 class ViewStyleCustomization {
-
+    /**
+     * Specify info panel peek height.
+     * Use [defaultInfoPanelPeekHeight] to reset to default.
+     */
+    @Px var infoPanelPeekHeight: Int? = null
```

Comment:
> I looked at this PR closer and seems like I can simply add Search views to InfoPanelBinder and return nulls in getHeaderLayout and getContentLayout, because I don't need that separation.

Correct. Returning `null` "tells" NavigationView that supplied layout does not support header and/or content binding, and that the application will manage those views.

> In addition to peek height, can we add an option to set bottom sheet state and to observe state changes?
Maybe we should simply expose BottomSheetBehavior.

I agree. However, this is not up to me. @abhishek1508

Reply:
Similar to `infoPanelPeekHeight` can we also add `infoPanelBackground` and `infoPanelBackgroundColor` to this file?

Reply:
> Similar to infoPanelPeekHeight can we also add infoPanelBackground and infoPanelBackgroundColor to this file?
That's up to you. I think `infoPanelBackgroundColor` is redundant. Developers can change it via `infoPanelBackground` drawable.

Reply:
Fine by me. Let's have `infoPanelBackground`